### DISCUSSION TITLE
got these errors : Source "forge-std/Script.sol" not found: File import callback not supported and Source "lib/forge-std/src/Script.sol" not found: File import callback not supported

### DISCUSSION THREAD

**Discussion by guyzilberblum** [Cyfrin/security-and-auditing-full-course-s23#166] [Q&A]
Im in section 3 in the course wrote a test and it gives me the error messages  Source "forge-std/Script.sol" not found: File import callback not supported 
Source "lib/forge-std/src/Script.sol" not found: File import callback not supported
i tried to do copy relative path but it did not worked

**Comment by EngrPips** [comment]
Hello @guyzilberblum, you resolved this already, right?

**Comment by guyzilberblum** [comment]
No still got it and I have another problem I think I need to jump on a call
with somebody 😂

**Reply by EngrPips** [reply]
Please show your test file and your working directory(open the lib folder)

**Comment by guyzilberblum** [comment]
Currently not at home is it ok if I send you tomorrow?

**Reply by EngrPips** [reply]
Yeah sure. Take your time.

**Comment by PatrickAlphaC** [comment]
@guyzilberblum have you gone through the basic foundry -> advanced foundry first? We teach how to overcome a lot of these problems.

It can be tempting to skip to the security stuff, but without a solid foundation you're going to run into a lot of issues!

**Comment by guyzilberblum** [comment]
Thanks, I'll check it out!

‫בתאריך יום ד׳, 1 במאי 2024 ב-6:39 מאת ‪Patrick Collins‬‏ <‪
***@***.***‬‏>:‬
